### DISCUSSION TITLE
Target audience

### DISCUSSION THREAD

**Discussion by jeromekelleher** [tskit-dev/what-is-an-arg-paper#20] [General]
It's important that we have a target audience in mind here so that we can communicate as effectively as possible with them.

To me, the people that we really want to communicate with here are the existing experts in the field - people who have already published on ARGs or are very familiar with them. Several recent papers have come out which are quite confused about what the output of tsinfer is and what (succinct) tree sequences are, and this demonstrates that the PIs in question don't understand what we're doing. So, we need to communicate directly with them as our primary and most important audience. 

As a consequence the paper should have the following properties:

1. Be short (these are busy people)
2. Get straight to the point, without unnecessary explanation or contextualisation
3. It should be possible (for, say, Nick Barton) to get the main points from just the figures and captions
4. Be published in a reputable technical journal (ideally Genetics, I guess, but maybe that's a different discussion)

Obviously it should be comprehensible to non-experts, but we're certainly not aiming at (say) interested undergrads or general biology audiences here.

**Comment by petrelharp** [comment]
This is sensible. To get them to read it, we'll need a different title than "What's an ARG", since they already (think they) know that. (And, probably we'll need to email it to them.) What's the hook to get them in?

**Reply by jeromekelleher** [reply]
Emailing is the right approach I think, I was imagining emailing a draft for comments to a bunch of people before preprinting.

You're right, the name needs to change probably...

So, the hook, good question. I'd say that understanding the relationship between node and edge annotated ARGs and that fact that we don't need recombination nodes for a lot of things is important?

**Reply by petrelharp** [reply]
I agree - but "understanding this is important" is not quite yet a hook. I think the dramatic difference in efficiency between the full ARG and the tree sequence is a hook? (and that's been pointed out elsewhere, eg the msprime paper, but this paper will explain it, so we can still use it in the title). Something like "Removing spurious information from the ARG for orders of magnitude speedup" but better would be an example? But, that probably doesn't capture most of what we want to get at here.

**Reply by jeromekelleher** [reply]
Good points - I'll ruminate a bit and open a discussion regarding the title when I have some thoughts.

**Comment by gregorgorjanc** [comment]
@jeromekelleher I think the audience is also folks that read about ARGs and tree sequences - because all of these concepts are thrown into kind-of the same bin, which makes it hard to follow what is what ;) So, aim at experts, but surely make it accessible to "entry" audience to max impact.

**Comment by hyanwong** [comment]
I wonder if we should actually be thinking of 2 papers, one succinct one for the technical audience, discussing the process vs outcome notation, node and edge annotated ARGs, simplification, unary nodes, etc (subtitle "Efficiency in ARG encodings: theory and practice", or whatever), and another on "What is an ARG", that references the previous paper, which will address @gregorgorjanc 's point. I also mention this because I hanker after a rigorous biological derivation of an ARG at the level of cell divisions, which I think would belong in the second sort of paper, as it is too much of an aside for the technical paper.

I think the audiences are reasonably distinct here, and the journals would probably be different, although the two papers can be, in some sense, complementary. Some of the discursive text I have written probably belongs more in the second type of paper than the first.

**Reply by jeromekelleher** [reply]
Yes, I think that's the right way to think about it. 

So, maybe we should come up with a better title for this paper? (In a different discussion?)

**Reply by hyanwong** [reply]
Should we (for the moment) attempt to keep both papers in this repo (and move material between them if necessary), or change the name of this repo and open another for the other paper? That is, assuming that other people agree that we want to have 2 papers.

**Reply by petrelharp** [reply]
The content destined for an "What is an ARG" paper might even be better suited for something on the website - less citeable, but more discoverable and probablywill be read more...

**Reply by jeromekelleher** [reply]
I'm strongly in favour of changing the name of this repo (once we've settled on a good title for the paper) and forgetting about paper 2 for now. We can write paper 2/website content later, but we need to get the technical paper showing the relationship between tree sequences and ARGS out ASAP. It'll only be matter of time before some feels the need to write a "real" ARG file format or (less likely) library otherwise. We need to convince people that tskit is the ARG library that they want to build their inference methods around.

**Comment by hyanwong** [comment]
Title discussion in https://github.com/tskit-dev/what-is-an-arg-paper/discussions/30

NB: looks like Alwyn Scally and myself would be interested in writing paper 2 after this one. Others free to join, of course. Perhaps I can get Alwyn to indoctrinate Cambridge into the wonderful world of tskit?
